docs/frontend_utilisation_guide.md

```diff
@@ -93,35 +143,66 @@ document.getElementById("myElementId").appendChild(spsApplication.rootElement);
+	if(webSocketAddress != ""){
+		config.setTextSetting(TextParameters.SignallingServerUrl, webSocketAddress)
+	}
 	const stream = new PixelStreaming(config);
-	const spsApplication = new SPSApplication({ stream });
+	const spsApplication = new SPSApplication({
+		stream,
+		onColorModeChanged: (isLightMode) => PixelStreamingApplicationStyles.setColorMode(isLightMode) /* Light/Dark mode support. */
+	});
 	document.body.appendChild(spsApplication.rootElement);
 }
 ```
 
-### Connecting to a WebSocket 
-When serving the Scalable Pixel Streaming Frontend it will build a default WebSocket address to connect to based on the address of the current window of the webpage. If the WebSocket address happens to match what is created by default then no further steps are required however, this is not always the case. In the stage of creating the `config` object for our entry point it is possible to inject a WebSocket address that will override the default that is created:
-
-1) After completing `Implementing the Scalable Pixel Streaming frontend in a custom webpage` steps 1-8 import the `TextParameters` package from the Pixel Streaming Frontend
+### Customising the WebSocket connection
+#### Using setTextSetting within Config to inject a custom websocket
+When serving the Scalable Pixel Streaming Frontend it will build a default WebSocket address to connect to based on the address of the current window of the webpage. If the WebSocket address happens to match what is created by default then no further steps are required however, this is not always the case. Through the usage of `setTextSetting` method on our `config` instance a user can inject a WebSocket address that will override the default. Steps to achieving this are covered in Basics to initialising and consuming the library steps 3 and 6.
+#### The .env file for the TypeScript example
+In the TypeScript example there is a `.env.example` file. inside this file there is a line called `WEBSOCKET_URL` containing a filler url. This file can be used to hard code a websocket address that can be consumed by the example as shown above. This example is able to work with the help of the [dotenv npm package](https://www.npmjs.com/package/dotenv) in the `webpack.common.js` file in the Typescript example. To implement this example steps:
+1) Rename the `.env.example` to `.env`
+2) Replace the place holder url with the websocket url you wish to consume
+3) Rebuild the example with the `npm run build-dev` or `npm run build-prod` for the changes to take effect
+
+If you wish to include this functionality in your project you will need to include the following steps:
+The TypeScript example makes use of these exact steps and is a good demonstration resource on this topic. 
+1) Install `dotenv` via npm `npm i dotenv --save-dev`
+2) Include `dotenv` in you webpack file and set your `.env` file path using `path:`
```

Comment:
change:
> in you webpack

to:
> in your webpack

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6